### PR TITLE
Add category tags

### PR DESCRIPTION
**Pull request by ctcampbell**
Add category tags for OWASP Top 10, or any other mapping via various new Xpath options

### CODE REVIEW

**Review by austenstone — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -64,13 +68,21 @@ try {
 try {
   cweXml = new DOMParser().parseFromString(readFileSync(cweFilePath, 'utf8'))
 } catch (err) {
+  console.log(`Unable to load CWE file: ${err}`)
```
Comment (austenstone) [inline src/main.ts:71]: `core.warning` or `core.info`

File: package.json [hunk 2]
```diff
@@ -27,23 +27,24 @@
   "author": "GitHub, Inc",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@xmldom/xmldom": "^0.8.7",
-    "jsonpath-plus": "^7.2.0",
-    "xpath": "^0.0.32",
+    "@actions/core": "^1.10.1",
+    "@xmldom/xmldom": "^0.8.10",
```
Comment (austenstone) [inline package.json:31]: `npm install -S @types/xmldom`

File: src/main.ts [hunk 3]
```diff
@@ -10,41 +10,45 @@ import * as xpath from 'xpath'
 import {JSONPath} from 'jsonpath-plus'
 
 let sarifFilePath: string
-let cweFilePath: string
-let cweIdXpath: string
-let securityStandardTag: string
 let outputFilePath: string
 let sarifResults: Object
```
Comment (austenstone) [inline src/main.ts:14]: `object` not `Object`

File: src/main.ts [hunk 4]
```diff
@@ -10,41 +10,45 @@ import * as xpath from 'xpath'
 import {JSONPath} from 'jsonpath-plus'
```
Comment (austenstone) [inline src/main.ts:10]: Remove `/* eslint-disable no-console */` at top

Comment (ctcampbell) [reply src/main.ts:10]: This isn't written for exclusive use in Actions so it does provide some console output.

File: package.json (hunk 2 above)
Comment (ctcampbell) [reply package.json:31]: xmldom includes type information in the package

File: src/main.ts (hunk 1 above)
Comment (ctcampbell) [reply src/main.ts:71]: There is already a message passed as part of core.setFailed()

**Review by aegilops — APPROVED** [review]

**Review by aegilops — APPROVED** [review]

**Pull request merged**